os/android/drmhwctwo.cpp

```diff
@@ -263,9 +263,10 @@ HWC2::Error DrmHwcTwo::HwcDisplay::GetChangedCompositionTypes(
 
 HWC2::Error DrmHwcTwo::HwcDisplay::GetClientTargetSupport(uint32_t /*width*/,
                                                           uint32_t /*height*/,
-                                                          int32_t /*format*/,
+                                                          int32_t format,
                                                           int32_t dataspace) {
-  if (dataspace != HAL_DATASPACE_UNKNOWN &&
+  if (!(display_->CheckPlaneFormat(format)) &&
```

Comment:
https://source.android.com/devices/halref/hwcomposer2_8h_source.html line 1053, "* This function must return true for a client target with width and height
 1066  * equal to the active display configuration dimensions,
 1067  * HAL_PIXEL_FORMAT_RGBA_8888, and HAL_DATASPACE_UNKNOWN."

We should have this as first check and than the plane format checks, dataspace checks like now.